editions/tw5.com/tiddlers/_tw_shared/styles.tid

```diff
@@ -1,5 +1,6 @@
 title: $:/_tw_shared/styles
 tags: $:/tags/Stylesheet TiddlyWikiSitesMenu
+refresh-blocker-list: [all[tiddlers]prefix[$:/state/]] [all[tiddlers]prefix[$:/temp/]] [all[tiddlers]is[draft]]
```

Comment:
IMO filter can be optimized for all that fields

Reply:
will do!